pkg/volume/gce_pd/gce_util.go

```diff
@@ -122,9 +137,111 @@ func (util *GCEDiskUtil) DetachDisk(pd *gcePersistentDisk) error {
 	if err := gce.(*gce_cloud.GCECloud).DetachDisk(pd.pdName); err != nil {
 		return err
 	}
+
+	devicePaths := []string{
+		path.Join("/dev/disk/by-id/", "google-"+pd.pdName),
```

Comment:
These are cut-n-pasted literal strings from AttachAndMountDisk().  Please factor out the code, or move to named constants.